documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/sessions/session-architecture.adoc

```diff
@@ -13,18 +13,31 @@
 
 [[pg-server-session-architecture]]
 ==== Session Architecture
+==== Introduction
+Jetty session support has been architected to provide a core implementation that is independent of the servlet specification.
+This allows programmers who use core Jetty - without the servlet api - to still have classic servlet session-like support for their `Requests` and `Handlers`.
```

Comment:
`api` -> `APIs`

`Requests` -> ``Request``s as the class does not have a final `s`. Same for `Handlers` and blow `Servlets`, `Filters`, etc.